src/components/Nav/Navbar.js

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import styled from "styled-components";
+import Burger from "./Burger";
+import "./navbar.css";
+
+const Nav = styled.nav`
+	width: 100%;
+	height: 80px;
+	border-bottom: 2px solid #f1f1f1;
+	padding: 1rem 1.2rem;
+	display: flex;
+	align-items: center;
+	background-color: black;
+	justify-content: space-between;
+	@media (max-width: 992px) {
+		justify-content: space-between;
+		gap: 1rem;
+		padding: 1rem 0;
+	}
+	.logo {
+		padding: 15px 0;
+	}
+`;
+
+const Navbar = () => {
+	return (
+		<Nav>
+			<Link to="/">
+				<div className="icon">
+					<img src="./images/logo.svg" alt="logo" />
```

Comment:
Great implementation of the Navbar Amin. Show we maybe have more descriptive alt text for the images? Lets not forget about a11y.

Reply:
Oh you're right! For thge accessibility I mentioned Ux designers to remove search bar from the navabr.